src/Relay/Relay/Cmdlets/Namespace/SetAzureRelayNamespace.cs

```diff
@@ -23,6 +24,7 @@ namespace Microsoft.Azure.Commands.Relay.Commands.Namespace
     /// <summary>
     /// 'Set-AzRelayNamespace' Cmdlet updates the specified Relay Namespace
     /// </summary>
+    [GenericBreakingChange("Set-AzRelayNamespace will be removed in an upcoming breaking change release, you can use Update-AzRelayNamespace in a future release")]
```

Comment:
Please add a target date or target version in breaking change message.

Reply:
Done